google-cloud-spanner/src/main/java/com/google/cloud/spanner/connection/ConnectionOptions.java

```diff
@@ -719,6 +730,11 @@ static boolean parseReadOnly(String uri) {
     return value != null ? Boolean.parseBoolean(value) : DEFAULT_READONLY;
   }
 
+  static boolean parseRouteToLeader(String uri) {
```

Comment:
Are we using this method within unit tests? If yes, for consistency we can add the @VisibleForTesting annotation like other methods.

Reply:
This is **not** used in unit tests. So, I think it would be cleaner to remove the annotation from other methods, rather than adding it here. I'll take it up as a follow-up PR, as that would be unrelated to the changes here.